asf_core_data/getters/mcs_getters/get_mcs_installations.py

```diff
@@ -133,3 +152,64 @@ def get_processed_installations_data(
             processed_installations_file_name,
         ),
     )
+
+
+def find_most_recent_mcs_installations_batch(epc_version: str = "none"):
```

Comment:
Missing the return type
```suggestion
def find_most_recent_mcs_installations_batch(epc_version: str = "none") -> str:
```